javascript/javascript1/week2/main.js

```diff
@@ -0,0 +1,103 @@
+// Flight booking fullname function
+
+console.log(getFullName("khuram", "Ejaz", true));
+console.log(getFullName("john", "smith"));
+
+function getFullName(firstName1, lastName1, useFormalName) {
+  if (useFormalName == true && firstName1 !== "" && lastName1 !== "") {
+    return "Lord " + firstName1 + " " + lastName1;
+  } else if (
+    useFormalName == false &&
+    firstName1 !== null &&
+    lastName1 !== null
+  ) {
+    return firstName1 + " " + lastName1;
+  } else if (
+    useFormalName == null &&
+    firstName1 !== null &&
+    lastName1 !== null
+  ) {
+    return firstName1 + " " + lastName1;
+  } else {
+    console.log("Please enter the first name and last name");
+  }
+}
+
+// Event application
+
+function getEventWeekDay(x) {
+  let d = new Date();
+  let day = (d.getDay() + x) % 7;
+  var days = [
+    "Sunday",
+    "Monday",
+    "Tuesday",
+    "Wednesday",
+    "Thursday",
+    "Friday",
+    "Saturday",
+  ];
+  let todaysDay = days[day];
```

Comment:
Variable naming convention should be used, here you get the eventDay not the todaysDay. also don´t use generic names like x etc... be specific.